src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -61,8 +61,28 @@ public sealed class FormatHex : PSCmdlet
         [Parameter(ParameterSetName = "ByInputObject")]
         public SwitchParameter Raw { get; set; }
 
+        /// <summary>
+        /// Offset of bytes to start reading the stream from.
+        /// </summary>
+        [Parameter()]
+        public UInt32 Offset { get; set; }
```

Comment:
How about files which greater then 4GB in size? Is not it have to be 64-bit type?